### PR TITLE
Exponential decay & retry delay chooser

### PR DESCRIPTION
**Pull request by EamonNerbonne**

### CODE REVIEW

File: src/ProgressOnderwijsUtils/RetryDelayChooser.cs [hunk 1]
```diff
@@ -0,0 +1,37 @@
+﻿using System;
+
+namespace ProgressOnderwijsUtils
+{
+    /// <summary>
+    /// This class pick a retry delay based on the current error rate (see ErrorsPerDayToRetryDelay for the formula)
```
Comment (tjibbechris) [inline src/ProgressOnderwijsUtils/RetryDelayChooser.cs]: Zin klopt niet

Comment (tjibbechris) [reply src/ProgressOnderwijsUtils/RetryDelayChooser.cs]: Perhaps more descriptive on the purpose of this class

Comment (EamonNerbonne) [reply src/ProgressOnderwijsUtils/RetryDelayChooser.cs]: updates.  Better?

File: src/ProgressOnderwijsUtils/RetryDelayChooser.cs [hunk 2]
```diff
@@ -3,8 +3,15 @@
 namespace ProgressOnderwijsUtils
 {
     /// <summary>
-    /// This class pick a retry delay based on the current error rate (see ErrorsPerDayToRetryDelay for the formula)
-    /// The error rate is estimated using exponential decay, so
+    /// This class picks a retry delay appropriate for a possibly intermittently failing service or similar process.
```
Comment (tjibbechris) [inline src/ProgressOnderwijsUtils/RetryDelayChooser.cs]: Prima

**Pull request merged**